includes/apple-exporter/class-parser.php

```diff
@@ -129,4 +134,56 @@ private function _parse_markdown( $html ) {
 		 */
 		return apply_filters( 'apple_news_parse_markdown', $content, $nodes );
 	}
+
+	/**
+	 * Handles cleaning up any HTML issues prior to parsing that could affect
+	 * both HTML and Markdown format.
+	 *
+	 * @access private
+	 * @param string $html
+	 * @return string The clean HTML
+	 */
+	private function _clean_html( $html ) {
+		// Match all <a> tags via regex.
+		// We can't use DOMDocument here because some tags will be removed entirely.
+		preg_match_all( '/<a(?:.*?)(href|name)=(?:"|\')(.*?)(?:"|\')(?:.*?)>(.*?)<\/a>/m', $html, $a_tags );
```

Comment:
Do you need that m modifier here? Isn't that only applicable if you're using `^` or `$` in the regex?